modules/asm/variables.tf

```diff
@@ -54,11 +54,65 @@ variable "service_account_key_file" {
 variable "asm_version" {
   description = "ASM version to deploy. Available versions are documented in https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages"
   type        = string
-  default     = "1.8"
+  default     = "1.9"
```

Comment:
note: breaking change, this will result in delete and recreate

Reply:
Maybe notice it in the release note ?
Should version change need to execute an upgrade instead ? 

We can let "1.8" but we need to explain the risk and the workaround to upgrade or downgrade the version. I didnt try it.